README.md

```diff
@@ -1,2 +1,89 @@
-# recombee
-Recombee API client.
+# Recombee
+
+A [Recombee API](https://docs.recombee.com/api.html) client written in Go. 
+
+[![GoDoc][GoDoc-Image]][GoDoc-Url]
+
+[GoDoc-Url]: https://pkg.go.dev/github.com/nats-io/nats.go
```

Comment:
?? Put there real reference not nats one

Reply:
My bad. Copy & paste.